<launch>

    <!-- It is often easier to start the servers separately then launch this file -->
    <arg name="is_launch_server" default="false" />
    <group if="$(arg is_launch_server)">
        <include file="$(find cordial_manager)/launch/start_servers.launch"/>
    </group>

    <param name="aws/region_name" type="str" value="us-west-1"/>
    <param name="cordial/speech/aws/voice_name" type="str" value="Ivy"/>
    <param name="cordial/speech/viseme/play_speed" type="double" value="10"/>
    <param name="cordial/speech/viseme/min_duration_in_seconds" type="double" value="0.05"/>
    <param name="cordial/speech/viseme/publish_delay_in_seconds" type="double" value="0.1"/>

    <node pkg="cordial_manager" type="manager.py" name="manager" output="screen"/>

    <include file="$(find cordial_sound)/launch/sound.launch" />
    <include file="$(find cordial_gui)/launch/gui.launch" />

</launch>
